examples/rag/state_of_the_union_rag.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+from dotenv import load_dotenv
+
+from mirascope.base import BaseChunker
+from mirascope.base.types import BaseChunkerParams
+from mirascope.chroma.vectorstores import ChromaVectorStore
+from mirascope.openai import OpenAICall, OpenAIEmbedder
+
+load_dotenv()
```

Comment:
pydantic-settings?

Reply:
i can convert if we want to keep it, but i was thinking about using an example different from langchain

Reply:
What do you mean? Do they use pydantic-settings?

I think regardless we should use pydantic-settings in our examples or directly set `os.environ["API_KEY"]`

Reply:
this example isnt the final one thats going to be pushed to dev. e.g. the state of the union, so I was going to scrap this once RAG is complete. the final one will use pydantic settings